plugins/catalog-backend-module-gitlab/src/lib/client.ts

```diff
@@ -136,16 +236,26 @@ export class GitLabClient {
         continue;
       }
 
-      memberIds.push(
-        ...response.data.group.groupMembers.nodes
-          .filter(n => n.user)
```

Comment:
I want to call out the deletion of this one. I vaguely remember it being added for very good reason - it could be the case that the user was null sometimes in the response, for some corner cases. You may want to retain that check, right

Reply:
Let me take another look here. This rings a bell, think I brought up this issue in https://github.com/backstage/backstage/issues/17735 and it slipped my mind

Reply:
Hi @freben, would adding this check here work ?https://github.com/backstage/backstage/pull/18889/commits/739b927df9cdb11bc8ae04dbc1b16cb3334afebf

Reply:
Hi @freben, would adding this check here work ?https://github.com/backstage/backstage/pull/18889/commits/739b927df9cdb11bc8ae04dbc1b16cb3334afebf

Reply:
Almost. I think it needs to be `group?.id` and `user.user?.id` or so to not throw an error